### DISCUSSION TITLE
about  Mag combined with model

### DISCUSSION THREAD

**Discussion by rainbownmm** [amazon-science/chronos-forecasting#145] [Q&A]
Thanks for your great work mag. The chronos model performs very well on my dataset. I wonder if you could provide me with an example of how to train adapters for specific tasks using mag as discussed in your paper. Thanks a lot.

**Comment by rainbownmm** [comment]
Also, if I want to combine mag, do I need to retrain the model or fine-tune it? Or train the mag model separately?

**Comment by abdulfatir** [comment]
What is MAG?

**Comment by rainbownmm** [comment]
It is about the multimodal paper mentioned in your paper
------------------------
Wasifur Rahman, Md Kamrul Hasan, Sangwu Lee, Amir Zadeh, Chengfeng Mao, Louis-Philippe Morency, and Ehsan Hoque. Integrating multimodal information in large pretrained transformers. In Proceedings of the conference. Association for Computational Linguistics. Meeting, volume 2020, pp. 2359. NIH Public Access, 2020. 19
